RunWorkFlow_CSHORE.py

```diff
@@ -126,25 +131,41 @@ def master_CSHORE_run(inputDict):
     # os.chdir(workingDir)  # is this right?  NO it's not
     curdir = os.getcwd()
 
+    # figure out which days need to be reset days
```

Comment:
Are you talking about the change to check to see which days need to be reset?  or tagging the current working directory as a variable?

Reply:
We made the change to check the reset days beforehand to avoid having to pull down the full bathy record on every model run.  Now mobile_reset and mobile just look for the previous days run (which is much faster), unless they are specifically told to reset.

and CSHORE_analysis requires the curdir variable to be assigned so it know which folder to go into to look for the output files.